packit/config/package_config.py

```diff
@@ -229,6 +230,21 @@ def get_copr_build_project_value(self) -> Optional[str]:
             )
         return projects_list[0]
 
+    def validate(self):
```

Comment:
I think we can do better in terms of the user experience: the problem with this solution is that if e.g. one of the files is missing, the code would halt there and wouldn't reach the next section - I'd prefer to get all error at once, not just one by one.

I'd therefore define a new class (e.g. PackageConfigValidation) which would hold data for all the fields if they are valid or not - if not, there would be an error message for the user saying what's wrong.

Reply:
Sure. Thank you. I'll look into it.